mlflow/store/tracking/file_store.py

```diff
@@ -332,10 +332,11 @@ def search_experiments(
 
     def get_experiment_by_name(self, experiment_name):
         def pagination_wrapper_func(number_to_get, next_page_token):
+            escaped_experiment_name = experiment_name.replace("'", "''")
```

Comment:
Can we directly check if `'` exist in experiment_name and raise error if it exists?

Reply:
BTW please add a unit test after the fix, thanks!

Reply:
Ok , Noted.